src/components/organisms/AssetActions/index.tsx

```diff
@@ -85,5 +86,9 @@ export default function AssetActions(): ReactElement {
       }
     )
 
-  return <Tabs items={tabs} className={styles.actions} />
+  return (
+    <Permission eventType="consume">
```

Comment:
not sure this is intended but this will not just disable consume but also the pool and trade tab and price display and compute. If you only want to disable consume, I would suggest rather removing the actual consume button

Reply:
Yeah, that is intentional. This is following the specification shared by @dungeonhao 

Only users who have consume access should be able to:  
        ◦ Add Liquidity
        ◦ Trade
        ◦ Compute
        ◦ Download 

Reply:
ok, just making sure it's clear you are locking people out of all financial data for a dataset with this, not just the actual actions. Same goes for people who have added liquidity and then were added to some restriction list, preventing them from removing liquidity

Reply:
This may be ok because this will be used in the enterprise world where there will be stricter conditions/contracts/rules.  But at the end of the day you are just locking the UI, because you can't prevent the user from interacting with the pools at an api level.

Reply:
@kremalicious It's a good point, I think the decision around that needs to come from the Daimler team though (@dungeonhao). If they are happy with this level of strictness or not. 

@mihaisc To my understanding we will also be implementing backend integrations with the RBAC server to stop the users interacting with the pools at API level.

Reply:
that means it has to be done at a contract level and i doubt it, but maybe i'm wrong